airbyte-integrations/connectors/source-notion/source_notion/streams.py

```diff
@@ -313,6 +330,20 @@ def stream_slices(
 
             yield {"page_id": page_id}
 
+    def transform(self, record: Mapping[str, Any]):
+        transform_object_field = record["type"]
```

Comment:
let's use `.get()` + default value, instead of the directly calling for the fields like `type`, etc.

Reply:
updated